camayoc/tests/qpc/cli/test_sources.py

```diff
@@ -46,11 +46,12 @@
 
 VALID_SOURCE_TYPE_HOSTS_WITH_EXCLUDE_HOSTS = (
     ('network', '192.168.0.42', '192.168.0.43'),
-    ('network', '192.168.0.1 192.168.0.2', '192.168.0.2 192.168.0.45'),
-    ('network', '192.168.0.0/24', '192.168.1.0/28'),
-    ('network', '192.168.0.[1:100]', '192.168.30.[1:100]'),
+    ('network', '192.168.0.42', '192.168.0.2 192.168.0.45'),
```

Comment:
Changing this you are changing the input data for the add tests. Maybe you need a new set for editing purposes?